addon/components/ember-table-row.js

```diff
@@ -75,6 +76,13 @@ export default class EmberTableRow extends Component {
   }
 
   click(event) {
-    this.get('targetObject').send('onRowClicked', event, this.get('rowIndex'));
+    const tableObject = this.get('row.api.targetObject');
+    tableObject.send('onRowClicked', event, this.get('rowIndex'));
+
+    this.sendAction('onRowClicked', event, this.get('rowIndex'), this.get('rowValue'));
```

Comment:
I think `onRowClicked` here is a bit verbose, from the user standpoint they already know they're passing in actions to the row. Let's do `onClick` instead